org.freedesktop.Sdk.Extension.qt5.yaml

```diff
@@ -0,0 +1,790 @@
+id: org.freedesktop.Sdk.Extension.qt5
+branch: '21.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk//21.08
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm13
+  - org.freedesktop.Sdk.Extension.node16
+build-extension: true
+build-options:
+  prefix: /usr/lib/sdk/qt5
+  append-path: /usr/lib/sdk/qt5/bin:/usr/lib/sdk/llvm13/bin:/usr/lib/sdk/node16/bin
+  append-pkg-config-path: /usr/lib/sdk/qt5/lib/pkgconfig
+  append-ld-library-path: /usr/lib/sdk/qt5/lib
+  prepend-ld-library-path: /usr/lib/sdk/llvm13/lib
+  cflags: -g0
+  cxxflags: -g0
+separate-locales: false
+appstream-compose: false
+modules:
+  - name: qt-base
+    buildsystem: autotools
+    config-opts:
+      - -confirm-license
+      - -opensource
+      - -shared
+      - -platform
+      - linux-g++
+      - -optimized-qmake
+      - -release
+      - -nomake
+      - tests
+      - -system-harfbuzz
+      - -system-sqlite
+      - -accessibility
+      - -dbus-linked
+      - -fontconfig
+      - -glib
+      - -icu
+      - -openssl-linked
+      - -no-rpath
+      - -no-directfb
+      - -no-kms
+      - -system-proxies
+      - -gtk
+      - -no-use-gold-linker
+    sources:
+      - type: git
+        url: git://code.qt.io/qt/qtbase.git
+        tag: v5.15.3-lts-lgpl
```

Comment:
Note that Qt company stopped making new releases of v5.15 for general public over one year ago so this version is abandoned. 5.15 is still supported by KDE who cherry-picks patches there.

Reply:
I don't see much of a point for development with the fork. So then the applications people would build with them would have to be linked to the fork and I'm assuming most distros do not ship that fork, limiting it's usefulness. It's not just for development of apps for flathub or using the KDE runtime in flatpak in general (in which case it would be a good idea though).

Reply:
At least ArchLinux and Fedora use fork. I don't know about other distros but most would prefer something active unless they're itself in some maintenance mode. Same question is for flathub - should it provide supported or unsupported software versions?

Reply:
Fedora does not use a (or the KDE) fork:
 https://src.fedoraproject.org/rpms/qt5-qtbase/blob/rawhide/f/qt5-qtbase.spec

they use patches where necessary.

Reply:
They [extract all patches from KDE fork](https://src.fedoraproject.org/rpms/qt5-qtbase/blob/f36/f/qt5-qtbase.spec#_142) and apply them on top of pristine Qt release which essentially means they use the fork. Do we really need to discuss about facts here?